<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IJ0KABL" failures="2" tests="4" name="Test" time="16.932" errors="0" timestamp="2025-07-08T08:39:17 IST">
  <testcase classname="api.testcases.UserTest" name="testCreateUser" time="6.074"/>
  <testcase classname="api.testcases.UserTest" name="testGetUseData" time="3.453">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.testcases.UserTest.testGetUseData(UserTest.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testGetUseData -->
  <testcase classname="api.testcases.UserTest" name="testUpdateUser" time="2.718"/>
  <testcase classname="api.testcases.UserTest" name="testDeleteUser" time="2.313">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.testcases.UserTest.testDeleteUser(UserTest.java:102)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
</testsuite> <!-- Test -->
